src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+ 
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+  
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+  padding: 0 50px;
+}
+  
+.nav_list {
+  display: flex;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+}
+
+.links {
+  margin-right: 20px;
+  position: relative;
+  list-style: none;
+  justify-content: center;
+  text-align: center;
+  padding: 23px 0;
```

Comment:
set this paddings to the `a`

Reply:
as i mentioned previously, when i set it to the a , all paddings had been worked wrongly

Reply:
why? they should work correctly, if you do everything in the right way

Reply:
and if you have questions, ask in the chat